src/actions/index.js

```diff
@@ -0,0 +1,18 @@
+export const ACTION_SIGN_IN = 'sign_in';
+export const ACTION_SIGN_UP = 'sign-up';
```

Comment:
please unify the action description to use all underscore like sign_up, contact_us.. etc